### ISSUE TITLE
List of links that should be supported by the app

### ISSUE THREAD

**Issue by jonan** [jonan/ForkHub#93]
## GitHub
- [x] https://github.com/jonan
- [x] https://github.com/jonan/ForkHub
- [x] https://github.com/jonan/ForkHub/issues
- [x] https://github.com/jonan/ForkHub/pulls
- [x] https://github.com/jonan/ForkHub/issues/1
- [x] https://github.com/jonan/ForkHub/commit/22b75b5a25ff8c8d03d1c047fd01a2fd598ae07d
- [ ] https://github.com/jonan/ForkHub/blob/master/CHANGELOG.md
- [ ] https://github.com/jonan/ForkHub/blob/3a638595811642c4ac2b7c5f0c5fc2a96c500fbd/CHANGELOG.md

## Gist
- [x] https://gist.github.com/jonan/2dbd9521a6307dfb38f0
- [x] https://gist.github.com/2dbd9521a6307dfb38f0

**Comment by RandomlyKnighted** [comment]
What about links to a comment of a diff? I know those sometimes appear in the news feed.

**Comment by jonan** [comment]
@RandomlyKnighted you're right, I've added all the links to specific parts of an issue that I've found.

**Comment by RandomlyKnighted** [comment]
@jonan Looks good!

**Comment by RandomlyKnighted** [comment]
Also, what about wiki links?

**Comment by Fuzzyma** [comment]
Note that also the blacklink is broken after the click on a link brought you to the Repo.  The blacklink points to the user instead of the issue where I clicked on that link

**Comment by jonan** [comment]
@Fuzzyma are you talking about your phone's back button or the back button in the top left hand corner?

**Comment by Fuzzyma** [comment]
I use the button on the top left corner

**Comment by jonan** [comment]
That's expected behavior, you are using the Up button, not the Back one. They are not exactly the same.

Checkout this link for more information http://developer.android.com/design/patterns/navigation.html

**Comment by moshpirit** [comment]
Still missing a changelog link too. It's kind of annoying having to read them in a browser

**Comment by KOLANICH** [comment]
@jonan, inability to open a file by link is very annoing. Especially taking in account that ForkHub can open files manually. Please add file links parsing as soon as possible. I gguess that all that you need to do it is to split path by slashes, check if third piece is ```blob```, treat the next one as a branch or a commit, switch to file tree tab, switch to that branch and simulate user actions to open that file.
